charts/sentry/values.yaml

```
@@ -2196,10 +2196,30 @@ zookeeper:
   # persistence:
   #   size: 8Gi
 
+redpanda:
+  enabled: true
+  fullnameOverride: "sentry-redpanda"
+  config:
+    cluster:
+      auto_create_topics_enabled: true
+  statefulset:
+    replicas: 1
+  listeners:
+    kafka:
+      port: 9092
+      tls:
+        enabled: false
+      authenticationMethod: "plaintext"
+  console:
+    enabled: false
+  auth:
+    sasl:
+      enabled: false
```

```suggestion
  auth:
    sasl:
      enabled: false
  external:
    enabled: false
  logging:
    usageStats:
      enabled: false
```
I suggest disabling external access as I don't think it is necessary and is best left disabled as a security measure.

Also, usage stats sent to Redpanda are [enabled by default](https://github.com/redpanda-data/redpanda-operator/blob/371f59cfe2cb4605c274c28dff56a29446a67f62/charts/redpanda/values.yaml#L305).